plugins/modules/gitlab_project_badge.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Guillaume MARTINEZ (lunik@tiwabbit.fr)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: gitlab_project_badge
+short_description: Manage project badges on GitLab Server
+version_added: 6.0.0
+description:
+  - This module allows to add and remove badges to/from a project.
+author: Guillaume MARTINEZ (@Lunik)
+requirements:
+  - C(owner) or C(maintainer) rights to project on the GitLab server
+extends_documentation_fragment:
+  - community.general.auth_basic
+  - community.general.gitlab
+
+options:
+  project:
+    description:
+      - The name (or full path) of the GitLab project the badge is added to/removed from.
+    required: true
+    type: str
+
+  state:
+    description:
+      - State of the badge in the project.
+      - On C(present), it adds a badge to a GitLab project.
+      - On C(absent), it removes a badge from a GitLab project.
+    choices: ['present', 'absent']
+    default: 'present'
+    type: str
+
+  link_url:
+    description:
+      - The URL associated with the badge
+    required: true
+    type: str
+
+  image_url:
+    description:
+      - The image URL of the badge
+    required: true
+    type: str
+
+notes:
+  - Supports C(check_mode).
```

Comment:
Instead of this:
```suggestion
```
...

Reply:
corrected by c4856f28490040f4f5274fbc90624ea7841df846